.rubocop.yml

```diff
@@ -1,205 +1,212 @@
+---
 AllCops:
   DisplayCopNames: true
-  TargetRubyVersion: 2.5
   Exclude:
-    - '*/vendor/**/*'
-    - '*/bin/**/*'
-    - '**/tmp/**/*'
-    - '*/spec/fixtures/**/*'
-
+  - "*/vendor/**/*"
+  - "*/bin/**/*"
+  - "**/tmp/**/*"
+  - "*/spec/fixtures/**/*"
+  TargetRubyVersion: 2.5
```

Comment:
unrelated: Ruby 2.5 is now EOL. Should we consider updating this to 2.6? https://www.ruby-lang.org/en/downloads/branches/

Reply:
I tried `2.7` when I saw it in the diff, but that nets a warning that it should match [spec.required_ruby_version](https://github.com/dependabot/dependabot-core/blob/dadd9bb5408604c633590de6675f751e0f0b46d0/common/dependabot-common.gemspec#L21)

```
dependabot-common.gemspec:21:32: C: Gemspec/RequiredRubyVersion: required_ruby_version (2.5, declared in dependabot-common.gemspec) and TargetRubyVersion (2.7, which may be specified in .rubocop.yml) should be equal.
  spec.required_ruby_version = ">= 2.5.0"
                               ^^^^^^^^^^
```

I was nervous about impacting https://rubygems.org/gems/dependabot-common for the linter. I'd also like this to be `2.7`, but that seemed like a big enough change to consider separately from this PR's scope of non-structural robot patches.